src/MOI_wrapper.jl

```diff
@@ -435,21 +435,42 @@ function LQOI.get_quadratic_terms_objective(model::Optimizer)
     return sparse(Int.(qmatind), qmatcol, qmatval)
 end
 
+"""
+Computes a conflict for the current model. This function should be used in case the model
+is infeasible, in order to determine a subset of the the constraints and of the variable
+that keep the model infeasible (this subset is called a conflict).
+
+Calling this function is required before trying to get any conflict attribute like
+`ConstraintConflictStatus` and `VariableConflictStatus`. Once a conflict is computed
+and the model is modified, the conflict is not automatically purged, and the old conflict
+is returned without a warning. 
+"""
 function compute_conflict(model::Optimizer)
     model.conflict = c_api_getconflict(model.inner)
     return
 end
 
+function _ensure_conflict_computed(model::Optimizer)
+    if model.conflict === nothing
+        error("Cannot access conflict status. Call `CPLEX.compute_conflict(model)` first. " * 
+            "In case the model is modified, the computed conflict will not be purged.")
```

Comment:
Minor thing: can you vertically align the `"` please. So
```julia
        error("Cannot access conflict status. Call `CPLEX.compute_conflict(model)` first. " * 
              "In case the model is modified, the computed conflict will not be purged.")
```